index.html

```diff
@@ -1,148 +1,181 @@
-          <li class="display-list-item"><a href="#" title="Visit my Medium"><img src="images/Median.jpeg" alt=""></a></li>
-          <li class="display-list-item"><a href="https://github.com/Wolfmikerson" title="Visit my Github" target="_blank" rel="noopener noreferrer"><img src="images/Github.jpeg" alt=""></a></li>
-          <li class="display-list-item"><a href="#" title="Visit my Angellist"><img src="images/Angellist.jpeg" alt=""></a></li>
+        <ul class="lan default-list-style first-list-style">
+          <li class="lang">Skills</li>
+          <li class=""><img src="images/right-brace.svg" alt="Skill"></li>
         </ul>
-		<button class="but default-list-style but1" type="submit" name="submit">Get my resume</button>
-		<ul class="lan ddefault-display-flex default-list-style second-list-tyle">
-			<li class="lang display-list-item">Languages</li>
-			<li class="display-list-item"><img src="images/Down-brace.svg" alt="Lang"></li>
-		</ul>
-		<div class="progL">
-			<ul class="pro default-list-style">
-				<li class="jsi"><img src="images/Ellipse.svg" alt="Jscript"></li>
-				<li class="js">JavaScript</li>
-			</ul>
-			<ul class="pro default-list-style">
-				<li class="jsi"><img src="images/Htm.svg" alt="Css"></li>
-				<li class="js">HTML</li>
-			</ul> 
-			<ul class="pro default-list-style">
-				<li class="jsi"><img src="images/css_logo.svg" alt="Css"></li>
-				<li class="js">CSS</li>
-			</ul>    
-		</div>
-		<div class="frame-skills">
-			<ul class="lan default-list-style first-list-style">
-				<li class="lang">Frameworks</li>
-				<li class=""><img src="images/right-brace.svg" alt="Frame"></li>
-			</ul>
-			<ul class="lan default-list-style first-list-style">
-				<li class="lang">Skills</li>
-				<li class=""><img src="images/right-brace.svg" alt="Skill"></li>
-			</ul>
       </div>
-	 </div>
-   </section>
-  </body>
-</html>
\ No newline at end of file
+    </div>
+  </section>
+  <!--- Footer Section-->
+  <footer class="footer headline default-display-flex">
+    <h2 class="contactme">Contact Me</h2>
+    <p class="form-description">If you have an application you are interested in developing, a feature that you need built or a
+      project that needs coding. I’d love to help with it</p>
+    <form class="form default-display-flex" action="https://formspree.io/f/xbjqpynb" method="post">
+      <input class="form-element" type="text" id="name" name="user_name" placeholder="Wolfmikerson" maxlength="30"
+        required>
+      <input class="form-element" type="email" id="mail" name="user_email" placeholder="wolfmikerson@gmail.com"
+        required>
+      <textarea class="form-element" id="msg" name="user_message" placeholder="Write your message here" maxlength="500"
+        required></textarea>
+      <button class="but but1 form-element" type="submit">get in touch</button>
+    </form>
```

Comment:
- [ ] It seems that there are some issues with your form, it does not look like the original one in Figma yet, see the difference below 👇 
- Your form
![image](https://user-images.githubusercontent.com/55465133/129964123-9fa35288-3944-415e-a7bc-23cea5b4fba3.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/129964205-d296e558-3cdc-4c52-b1f7-84c1aed6d18f.png)

Can you see the difference? please let us fix the issue to make the form look like the Figma design 😉 

- [ ] Kindly let us add Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. 👍